src/Components/BookCard/BookCard.css

```diff
@@ -6,34 +6,34 @@
 .bookcover {
   height: 390px;
   width: 100%;
-  border: none;
-  border: 3px solid black;
-  border-radius: 20px 20px 20px 20px;
-}
-
-.to-read-btn {
-  font-size: 20px;
+  /* border: none; */
+  /* border: 3px solid black; */
+  border-radius: 20px;
 }
 
 .bookdata {
   display: flex;
-  justify-content: space-around;
+  justify-content: space-between;
```

Comment:
I think that if we keep the border ( rather than taking it out for a more uniform appearance with different length book titles ) it would be better to extend the border to wrap around the image as well

Reply:
I agree with Naomi, if we keep a border it should visually connect the content to the book that it's describing. I think that it would look nice without the border but am flexible if you two prefer it.